resources/definitions/ultimaker_s7.def.json

```diff
@@ -9,12 +9,10 @@
         "manufacturer": "Ultimaker B.V.",
         "file_formats": "application/x-ufp;text/x-gcode",
         "platform": "ultimaker_s7_platform.obj",
-        "bom_numbers": [
-            5078167
-        ],
+        "bom_numbers": [5078167, 5078168],
         "firmware_update_info":
         {
-            "check_urls": [ "https://software.ultimaker.com/releases/firmware/9051/stable/um-update.swu.version" ],
+            "check_urls": [ "https://software.ultimaker.com/releases/firmware/5078167/stable/um-update.swu.version" ],
             "id": 5078167,
             "update_url": "https://ultimaker.com/firmware?utm_source=cura&utm_medium=software&utm_campaign=fw-update"
```

Comment:
This link doesn't show the S7 firmware

Reply:
It does, the current Stable firmware for the S7 (BoM number 5078167) is 8.1.1, which is what is reported by the contents of the file `um-update.swu.version`